app/assets/javascripts/notes.js

```diff
@@ -50,6 +50,8 @@ jQuery(document).ready(function($) {
     {
       addTag("date:"+eventDate.val());
       $('#mytoast').addClass("show").fadeIn('fast').delay(2000).fadeOut('fast');
+      $("#event-datepicker").hide();
++     $("#date-toggle").hide();
```

Comment:
@roli25 You should remove the plus sign, it's not part of the code.

Reply:
@TildaDares Thankyou for helping.